### DISCUSSION TITLE
Cloudstack Baremetal Setup

### DISCUSSION THREAD

**Discussion by Aashiqps** [apache/cloudstack#9606] [Q&A] [answered]
I have an existing cloudstack setup (ver 4.19.0.2) with 1 mangement server,  5 KVM hosts, Linstor clustor (1 combined and 2 satellite nodes) in dis-aggregate architecture as primary storage and an NFS secondary storage. Cloudstack is configured with Advanced Networking. I want to integrate a baremetal host into this current cloudstack environment, would like to know whether it is possible or not and requirements for a baremetal integration.

**Comment by matats** [comment]
Sorry to bump, but I have the same requirement - baremetal setup. I've searched the web for days and found no solution anywhere. Getting really desperate.
Thanx in advance

**Reply by zap51** [reply, marked as answer]
@matats, there seems to be a discussion at https://github.com/apache/cloudstack/discussions/10058
